react/src/FileUpload/utils/getReadableFizeSize.ts

```diff
@@ -0,0 +1,14 @@
+const DECIMAL_BYTE_UNITS = ['B', 'kB', 'MB', 'GB', 'TB']
+
+/**
+ * Converts the given file size in bytes to a human readable string.
+ *
+ * @example 1100000 -> "1.1 MB"
+ * @param fileSizeInBytes the size of the file in bytes to be converted to a readable string
+ * @returns the human-readable file size string
+ */
+export const getReadableFileSize = (fileSizeInBytes: number): string => {
```

Comment:
`getReadableFileSize(0)` returns `NaN undefined`, but Idk how far you want to go w.r.t. error handling cus setting `maxFileSize` as `0` is obv the OGP dev fault...

<img width="1620" alt="image" src="https://github.com/opengovsg/design-system/assets/14961285/281edd22-c0dc-4a8f-8165-0f061e71ecb1">

Reply:
And I suppose technology is not that advanced yet - 1000 TB gives `1 undefined`